### DISCUSSION TITLE
Operator version upgrade

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx-operator#503] [Q&A] [answered]
Hi, currently I am using emqx version 4 with operator version 1.2.2, I want to use emqx version 5 which has operator version 2.0.1.
If I upgrde operator version will it affect my emqx vesiion 4 deployments?

**Comment by Rory-Z** [comment, marked as answer]
You can just run `kubectl patch crd emqxbrokers.apps.emqx.io --type merge -p '{"spec":{"names":{"shortNames":[]}}}'` before install EMQX Operator 2.0, and we commend you can install 2.0.2, not 2.0.1, because it's more new

**Reply by alexa22013** [reply]
ok thanks, is there any documentation for emqx operator upgrade?

**Reply by Rory-Z** [reply]
Sorry, we don't have documentation about this, but we're working on that part and you'll see it soon
